packages/liveblocks-node/tsconfig.json

```diff
@@ -6,5 +6,5 @@
     "target": "es2020",
     "module": "es2020"
   },
-  "include": ["src/**/*"]
+  "include": ["src/**/*", "test-d"]
```

Comment:
TypeScript itself should not know about this dir I think. The way you run tsd is through `turbo run test:types`, which first builds to `dist/`, and tsd will run against the produced `dist/index.d.mts` file. Including it here will make TypeScript look at these files, which is not necessary I believe.

Reply:
Makes sense! and done!